asapdiscovery/simulation/scripts/MProFunction.py

```diff
@@ -0,0 +1,239 @@
+    # unit cell information so we just create a box that has a 9A padding around the complex.
+    modeller.addSolvent(system_generator.forcefield, model='tip3p', padding=0.9*unit.nanometers)
+    logger.info(':package:  System has %d atoms' % modeller.topology.getNumAtoms())
+
+    # Create the system using the SystemGenerator
+    logger.info(':globe_showing_americas:  Creating system...')
+    system = system_generator.create_system(modeller.topology, molecules=ligand_mol)
+
+    # Add virtual bonds so solute is imaged together
+    logger.info(f':chains:  Adding virtual bonds between molecules')
+    custom_bond_force = openmm.CustomBondForce('0')
+    for molecule_index in range(len(molecules_atom_indices)-1):
+        custom_bond_force.addBond(molecules_atom_indices[molecule_index][0], molecules_atom_indices[molecule_index+1][0], [])
+    system.addForce(custom_bond_force)
+
+    # Add barostat
+    system.addForce(MonteCarloBarostat(pressure, temperature))
+    logger.info(f':game_die: Default Periodic box:')
+    for dim in range(3):
+        logger.info(f'  :small_blue_diamond: {system.getDefaultPeriodicBoxVectors()[dim]}')
+
+    # Create integrator
+    logger.info(':building_construction:  Creating integrator...')
+    integrator = LangevinMiddleIntegrator(temperature, friction, dt)
+
+    # Create simulation
+    logger.info(':mage:  Creating simulation...')
+    simulation = Simulation(modeller.topology, system, integrator, platform=platform)
+    context = simulation.context
+    context.setPositions(modeller.positions)
+
+    # Minimize energy
+    logger.info(':skier:  Minimizing ...')
+    simulation.minimizeEnergy()
+
+    # Equilibrate
+    logger.info(':fire:  Heating ...')
+    simulation.context.setVelocitiesToTemperature(temperature)
+    simulation.step(equilibrationSteps)
+
+    #logger.info(':fencer: Creating New Output Directory')
+    #new_directory = os.mkdir(os.path.join(output_dir + save))
+
+    # Reporters
+    logger.info(':steam_locomotive: Running Simulation...')
+    dataReporter = simulation.reporters.append(StateDataReporter(os.path.join(output_dir + save + '_simulation_' +'log.txt'), 50, totalSteps=steps,
+        step=True, speed=True, progress=True, potentialEnergy=True, temperature=True, separator='\t'))
+    xtcReporter = simulation.reporters.append(XTCReporter(os.path.join(output_dir + save + '_simulation_' +'.xtc'), reportingInterval))
+
+    simulation.currentStep = 0
+    simulation.step(steps)
+
+    logger.info(':CD: Saving...')
+    positions = simulation.context.getState(getPositions=True).getPositions()
+    PDBFile.writeFile(simulation.topology, positions, open(os.path.join(output_dir + save + '_simulation' + '_final.pdb'), 'w'))
+    simulation.saveState(os.path.join(output_dir,'output.xml'))
+    print('Done')
+
+    def main():
+        args = get_args()
+
+        protein = Path(args.protein)
+        proteinpaths = protein.glob("*protein.pdb")
```

Comment:
so instead of this we want to do something like
```
input_dir = Path(args.input_dir)
proteinpaths = list(input_dir.glob(args.protein_glob))
ligandpaths = list(input_dir.glob(args.ligand_glob))
```

What you currently have will search for all files with a path like `path_to_protein/protein.pdb*protein.pdb` which won't work